### PR TITLE
#v4.3.2112.0-rc - Support for api error codes

### PR DESCRIPTION
**Pull request by ppatidar2021**

### CODE REVIEW

File: BitPay/Exceptions/SettlementException.cs [hunk 1]
```diff
@@ -14,5 +15,19 @@ public SettlementException() : base(BitPayCode, BitPayMessage)
         public SettlementException(Exception ex) : base(BitPayCode, BitPayMessage, ex)
         {
         }
+
+        public SettlementException(string bitpayCode, string message) : base(bitpayCode, message)
```
Comment (anbugal) [inline BitPay/Exceptions/SettlementException.cs:19]: Is this one neccesary?

Comment (ppatidar2021) [reply BitPay/Exceptions/SettlementException.cs:19]: I don't think, But i added because we have same method in others exceptions.

Comment (anbugal) [reply BitPay/Exceptions/SettlementException.cs:19]: in other you have only 2 constructors

Comment (ppatidar2021) [reply BitPay/Exceptions/SettlementException.cs:19]: Yes, there are 2 constructors but those are in crud exceptions  of models.
Parent Constructors of every model has more then 2 constructors.

**Pull request merged**